src/SprykerEco/Zed/ComputopApi/Business/Mapper/ComputopApiBusinessMapperFactory.php

```diff
@@ -66,471 +66,471 @@ public function createAuthorizeCreditCardMapper(): PostPlaceMapperInterface
         return new AuthorizeCreditCardMapper(
             $this->getComputopApiService(),
             $this->getConfig(),
-            $this->getStoreFacade()
+            $this->getStoreFacade(),
         );
     }
 
     /**
-     * @return \SprykerEco\Zed\ComputopApi\Business\Mapper\PostPlace\PostPlaceMapperInterface;
+     * @return \SprykerEco\Zed\ComputopApi\Business\Mapper\PostPlace\CreditCard\ReverseCreditCardMapper;
      */
-    public function createReverseCreditCardMapper(): PostPlaceMapperInterface
+    public function createReverseCreditCardMapper(): ReverseCreditCardMapper
```

Comment:
I'm not sure about these changes, why do you need them?

Reply:
Changed this because quality tools show error message for invalid return type